dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -203,8 +203,7 @@ public void testLinkageReports() {
     Nodes reports = details.query("//p[@class='jar-linkage-report']");
     // appengine-api-sdk, shown as first item in linkage errors, has these errors
     Truth.assertThat(trimAndCollapseWhiteSpace(reports.get(0).getValue()))
-        .isEqualTo(
-            "106 target classes causing linkage errors referenced from 516 source classes.");
+        .isEqualTo("106 symbols causing linkage errors referenced from 51 classes.");
```

Comment:
The symbol (target class) number remains same. This PR has enhancement to count unique source classes. Previously it was not counting unique number (screenshot in PR description).

Reply:
So are you claiming the previous 516 number was incorrect?

Reply:
Yes, 516 source classes was not unique. The screenshot in [the PR summary](https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/631#issue-282943824) explains the unique count should be 51 in old (current) dashboard.